updatecli/weekly.d/tfenv.yaml

```
@@ -0,0 +1,50 @@
+---
+version: 0.33.3
+title: "Update tfenv version"
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: {{ .github.user }}
+      email: {{ .github.email }}
+      owner: {{ .github.owner }}
+      repository: {{ .github.repository }}
+      token: {{ or .github.token (requiredEnv "UPDATECLI_GITHUB_TOKEN") }}
+      username: {{ or .github.user (requiredEnv "UPDATECLI_GITHUB_ACTOR") }}
+      branch: {{ or .github.branch "main" }}
+
+sources:
+  latestTfenvImageVersion:
+    kind: githubrelease
+    name: Get the latest published tfenv version
+    spec:
+      owner: tfutils
+      repository: tfenv
+      token: {{ or .github.token (requiredEnv "UPDATECLI_GITHUB_TOKEN") }}
+      versionfilter:
+        kind: semver
+
+targets:
+  setTfenvImageVersion:
+    sourceid: latestTfenvImageVersion
+    name: Bump Tfenv Version
+    kind: file
+    scmid: default
+    spec:
+      files:
+        - scripts/tfenv-install-terraform.sh
+      matchpattern: 'git clone -b v(\d+\.)?(\d+\.)?(\*|\d+)'
+      content: 'git clone -b v{{ source `latestTfenvImageVersion` }}'
+
+pullrequests:
+  setTfenvImageVersion:
+    kind: github
+    scmid: default
+    title: >-
+      [updatecli] Bump Tfenv version to {{ source "latestTfenvImageVersion" }}
```

it's tfenv not Tfenv (applies throughout)

```suggestion
      [updatecli] Bump tfenv version to {{ source "latestTfenvImageVersion" }}
```